src/mappings/vault.ts

```diff
@@ -151,7 +152,16 @@ export function handleInitialized(event: Initialized): void {
     vault.validatorsManager = DEPOSIT_DATA_REGISTRY
   }
 
+  if (vault.version.toString() === vault.osTokenConfig) {
+    const newOsTokenConfigVersion = newVersion.toString()
+
+    createOrLoadOsTokenConfig(newOsTokenConfigVersion)
+
+    vault.osTokenConfig = newOsTokenConfigVersion
```

Comment:
Looks like it will break once the vault version will change to 3, but the OsTokenConfig contract won't change.